### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by knockoffduck**
I've merged everyone's branches together, however please try to clone and run the branch first to ensure that everything is correct.

### CODE REVIEW

File: .gitinore
Comment (eli3005) [inline .gitinore:1]: Need to delete this file and add .vscode to the real one

File: README.md
Comment (eli3005) [inline README.md:1]: Need to structure the table scheme properly and add to descriptions of chat

File: change avatar
Comment (eli3005) [inline change avatar:1]: To be added as a function in tests

File: schema_generator.py
Comment (eli3005) [inline schema_generator.py:1]: To be deleted as not used. Just used in the readme

File: tests/test_selenium.py
Comment (eli3005) [inline tests/test_selenium.py:1]: Test_chat_functionality to be fixed

File: tests/unit/test_unit.py
Comment (eli3005) [inline tests/unit/test_unit.py:1]: Have to fix avatar test

**Review by eli3005 — APPROVED** [review]
I left some comments of things that need to be fixed. I’ll fix testing too

**Review by 23640056 — APPROVED** [review]

**Review by tom18072001 — APPROVED** [review]

**Review by tom18072001 — APPROVED** [review]

**Pull request merged**